amf-api-contract/shared/src/main/scala/amf/apicontract/internal/spec/oas/parser/domain/LicenseParser.scala

```diff
@@ -23,6 +24,16 @@ class LicenseParser(node: YNode)(implicit ctx: WebApiContext) extends SpecParser
     val map = node.as[YMap]
     map.key("url", LicenseModel.Url in license)
     map.key("name", LicenseModel.Name in license)
+    map.key("identifier", LicenseModel.Identifier in license)
```

Comment:
This parser is also for OAS3. The new field should be parsed and validated only for OAS3.1